# Request
sensor_msgs/JointState joint_state
---
# Response
bool status